apps/web/src/features/resources/components/DropdownMenu/resources-category-dropdown-menu.tsx

```diff
@@ -44,6 +47,32 @@ export const ResourcesDropdownMenu = ({
     };
   }, [ref]);
 
+  if (backToCategoryButton) {
+    return (
+      <div className={'w-fit max-w-[280px] mx-auto mb-6 md:hidden'}>
+        <Link to={`/resources/${resourceActiveCategory}`}>
+          <button
+            type="button"
+            className={cn(
+              'flex items-center gap-2 p-2 pr-3 w-full bg-darkOrange-11 rounded-xl border border-darkOrange-9',
+            )}
+          >
+            <CategoryIcon
+              src={activeCategoryImageSrc || ''}
+              variant="resources"
+              imgClassName="filter-darkOrange size-6"
+            />
+            <span className="text-darkOrange-5 font-medium leading-[140%] tracking-[0.15px]">
```

Comment:
I see tracking-[0.15px] in many placed in conference page.
Could we have "global" styles for this ?